crates/xline/src/storage/auth_store/store.rs

```diff
@@ -193,13 +193,13 @@ impl AuthStore {
     ) -> Result<CommandResponse, ExecuteError> {
         #[allow(clippy::wildcard_enum_match_arm)]
         let res = match *request {
-            RequestWrapper::AuthEnableRequest(ref req) => {
+            RequestWrapper::AuthEnableRequest(req) => {
                 self.handle_auth_enable_request(req).map(Into::into)
             }
-            RequestWrapper::AuthDisableRequest(ref req) => {
+            RequestWrapper::AuthDisableRequest(req) => {
                 Ok(self.handle_auth_disable_request(req).into())
             }
-            RequestWrapper::AuthStatusRequest(ref req) => {
+            RequestWrapper::AuthStatusRequest(req) => {
                 Ok(self.handle_auth_status_request(req).into())
             }
             RequestWrapper::AuthUserAddRequest(ref req) => {
```

Comment:
Why do some requests use `req` while others use `ref req`?

Reply:
Clippy indicated that it's more efficient to pass value than reference for `AuthStatusRequest`. The same reason applies to other similar changes.